### PR TITLE
[3.x] Add translation checker workflow

### PR DESCRIPTION
**Pull request by Jade-GG**
Adds the translation checker workflow. Will show missing translations for now as it's missing the translations coming from the core itself (those are only in 3.x), however that will be resolved once the composer require is set to 3.x

### CODE REVIEW

File: lang/nl.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "Register": "Registreren",
+    "Register account": "Account registreren",
+    "Register within 1 minute": "Registreer binnen 1 minuut",
+    "Return to home": "Terug naar home",
+    "Security token": "Beveiligingstoken",
+    "Select an address": "Selecteer een adres",
+    "Select as billing": "Selecteer als factuuradres",
+    "Select as shipping": "Selecteer als verzendadres",
+    "Send": "Versturen",
+    "Step 1 of 4": "Stap 1 van 4",
```
Comment (indykoning) [inline lang/nl.json:75]: Do we want this hardcoded like this?

File: lang/nl.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    "Order overview": "Bestellingsoverzicht",
+    "Ordered within 2 minutes": "Besteld binnen 2 minuten",
+    "Place repeat order / View orders": "Herhaal bestelling / Bekijk bestellingen",
+    "Please select an address": "Selecteer een adres",
+    "Price": "Prijs",
+    "Product": "Product",
+    "Quick and easy ordering without having to fill in your details every time": "Snel en eenvoudig bestellen zonder elke keer uw gegevens in te vullen",
+    "Register": "Registreren",
+    "Register account": "Account registreren",
+    "Register within 1 minute": "Registreer binnen 1 minuut",
```
Comment (indykoning) [inline lang/nl.json:68]: This is a USP, correct?

Comment (Jade-GG) [reply lang/nl.json:68]: Yup, however one that does exist in the source: https://github.com/rapidez/checkout-theme/blob/master/resources/views/account/partials/register.blade.php#L3

File: lang/nl.json (hunk 1 above)
Comment (Jade-GG) [reply lang/nl.json]: No, I've gone and fixed this now :)

File: resources/views/cart/partials/top-bar.blade.php [hunk 3]
```diff
@@ -1,10 +1,15 @@
+@php($checkoutSteps = config('rapidez.frontend.checkout_steps.' . config('rapidez.store_code')) ?: config('rapidez.frontend.checkout_steps.default'))
```
Comment (royduin) [inline resources/views/cart/partials/top-bar.blade.php:1]: Do we still needs this with: https://github.com/rapidez/core/blob/master/src/Http/Controllers/CheckoutController.php#L24, can't we use the variable from the controller?

Comment (Jade-GG) [reply resources/views/cart/partials/top-bar.blade.php:1]: Neat, definitely would replace that here.

Funny story, turns out this file isn't even used anymore so I'm deleting it instead 🙂

**Comment by royduin** [comment]
The workflow is failing now, is that correct?

**Comment by Jade-GG** [comment]
> The workflow is failing now, is that correct?

Correct, at this point the translations from the core are missing---those haven't been released yet (i.e. Rapidez 3.x hasn't been released yet).

Once those are released and the composer.json gets updated with 3.x, then we'll get the core translations merged in and the workflow will succeed 🙂

**Pull request merged**